.github/pull-request-template.md

```diff
@@ -4,9 +4,16 @@
 
 #### Description
 Describe your changes here.
+
+#### Release note
+REQUIRED: Enter a simple message for this commit which will go in the release notes. Include how this will affect a project or a developer.
+
 #### Tests
 How did you test these changes prior to submitting this pull request?
 What automated tests are included in this PR?
+
+REQUIRED: How can this be tested by QA?
```

Comment:
Not necessarily required, right? Some issues are not reasonable/feasible for them to test. STRONGLY SUGGESTED? :trollface:

Reply:
You're not wrong but I'd rather people consciously delete this rather than ignore by default :p

Reply:
But then it's not required? What do words mean then? true == false

Reply:
Fineeeeeeeeee